src/components/cars/newcar.js

```diff
@@ -1,7 +1,3646 @@
+  const [description, setDescription] = useState('');
+  const [rentFee, setRentFee] = useState('');
+  const [modelYear, setModelYear] = useState('');
+  const [, setMessage] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const car = {
+        brand,
+        model,
+        photoUrl,
+        rentFee,
+        modelYear,
+        description,
+
+      };
+
+      const res = await fetch('http://127.0.0.1:3000/api/cars', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(car),
+      });
+      await res.json();
+      if (res.status === 200) {
+        setBrand('');
+        setModel('');
+        setPhotoUrl('');
+        setRentFee('');
+        setModel('');
+        setDescription('');
+        setMessage('User created successfully');
+        navigate('/');
+      } else {
+        setMessage('Some error occured');
+      }
+    } catch (err) {
+      setMessage(err);
+    }
+  };
+
+  return (
+    <form
+      onSubmit={handleSubmit}
+      className="wrapper"
+      style={
+            { margin: '5% 30% 20% 30%', boxShadow: '0 0 10px', padding: '5%' }
+        }
+    >
+
+      <Form.Control
+        placeholder="Image"
+        onChange={
+            (e) => setPhotoUrl(e.target.value)
+        }
+        value={photoUrl}
+        type="text"
+        alt="image"
+        id="photo-url"
+        className="photo-url mt-3 mr-4 ml-4"
+      />
+      <Form.Control
+        placeholder="Brand"
+        onChange={
+            (e) => setBrand(e.target.value)
+        }
+        value={brand}
+        type="text"
+        id="brand"
+        className="brand mt-3"
+      />
+      <Form.Control
+        placeholder="Model year"
+        onChange={
+            (e) => setModelYear(e.target.value)
+        }
+        value={modelYear}
+        type="number"
+        id="modelYear"
+        className="model-year mt-3"
+      />
+      <Form.Control
+        placeholder="Description"
+        onChange={
+            (e) => setDescription(e.target.value)
+        }
+        value={description}
+        type="text"
+        id="description"
+        className="description mt-3"
+      />
+      <Form.Control
+        placeholder="Rent fee"
+        onChange={
+            (e) => setRentFee(e.target.value)
+        }
+        value={rentFee}
+        type="text"
+        id="rentFee"
+        className="rent-fee mt-3"
+      />
+      <Form.Control
+        placeholder="Model"
+        onChange={
+            (e) => setModel(e.target.value)
+        }
+        value={model}
+        type="text"
+        id="model "
+        className="model mt-3"
+      />
+      <button
+        type="submit"
+        className="btn btn-success btn-lg mt-5"
+        style={
+            { marginLeft: '35%' }
+        }
+      >
+        { ' ' }
+        {' '}
+        { ' ' }
+        {' '}
+        { ' ' }
```

Comment:
there is multiple empty spaces in your file. kindly removes these extra spaces to make the code concise

Also Kindly see how your project appears below.
![image](https://user-images.githubusercontent.com/43564324/161252298-74785467-f671-4cd3-a8f4-3963ca0914af.png)
desktop view

![image](https://user-images.githubusercontent.com/43564324/161252548-4e5dd508-c2de-42ff-8c25-7ffe492ede6a.png)
mobile view

Please make the form to appear in the center of the page

Reply:
I was all working fine perior to when I git pull from dev branch, there's probably a CSS clash, with my inline CSS, for the spaces that eslint, I will do without eslint so the task assigned will do that